consumers/jira_c/gojira/jirautils.go

```diff
@@ -0,0 +1,162 @@
+package gojira
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/andygrunwald/go-jira"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	// EnvConfigPath the path towards the config.yaml file
+	EnvConfigPath = "DRACON_JIRA_CONFIG_PATH"
+)
+
+type customField struct {
+	ID        string   `yaml:"id"`
+	FieldType string   `yaml:"fieldType"`
+	Values    []string `yaml:"values"`
+}
+
+type defaultValues struct {
+	IssueFields  map[string][]string `yaml:"issueFields,omitempty"`
+	CustomFields []customField       `yaml:"customFields,omitempty"`
+}
+
+type mappings struct {
+	DraconField string `yaml:"draconField"`
+	JiraField   string `yaml:"jiraField"`
+	FieldType   string `yaml:"fieldType"`
+}
+
+type config struct {
+	DefaultValues     defaultValues `yaml:"defaultValues"`
+	Mappings          []mappings    `yaml:"mappings"`
+	DescriptionExtras []string      `yaml:"addToDescription"`
+}
+
+// getConfig parses the configuration from config.yaml and returns a config struct
+func getConfig() config {
+	configFile, err := ioutil.ReadFile(os.Getenv(EnvConfigPath))
+	if err != nil {
+		log.Printf("Error while reading config file:   #%v ", err)
+	}
+
+	var newConfig config
+	err = yaml.Unmarshal(configFile, &newConfig)
+	if err != nil {
+		log.Fatalf("Unmarshal: %v", err)
+	}
+	return newConfig
+}
+
+// parseDraconMessage returns a hashmap of the parsed dracon message
+func parseDraconMessage(message string) (map[string]string, error) {
```

Comment:
Due to the way the config.yaml file affects the consumer, it makes more sense to represent Dracon Results as a HashMap rather than the Document object. Example:

If the user specifies in config.yaml that they want to append several fields to the description (e.g. 'severity_text', 'cvss', 'target',...) it's easy to just iterate over these keys and write the code as `Description += DraconResult[key]`.
However if you weren't using a HashMap, you wouldn't be able to access the Dracon Result fields by their name and would have to resort to something like `if key == 'cvss': Description += DraconResult.CVSS`, making the code more cluttered.

As a "middle ground" solution, I've improved the code in `utils.go` to always return Dracon Results as HashMaps (whereas before they were returned as strings), cutting out a layer of conversion.